website/docs/r/rde.html.markdown

```diff
@@ -185,6 +186,55 @@ should have `resolve=false` to avoid being resolved).
 In this last scenario, it is advisable to mark `resolve_on_removal=true` so Terraform can delete the RDE even if it was not
 resolved by anyone.
 
+<a id="metadata"></a>
+## Metadata
+
+-> Due to Terraform limitations, `terraform plan` shows that all metadata entries will be deleted and re-created when performing
+a change in of one or more of them, but applying it will respect every non-changed metadata entry by not performing any modification on
+these.
```

Comment:
I'm trying to simplify and clarify the statement:

```suggestion
-> Due to Terraform limitations, when performing a change in any of the metadata entries, a `terraform plan` shows that all of them will be deleted and re-created. However, the succeeding `terraform apply` will respect the non-changed metadata entries by not modifying them.
```

Reply:
Yeah, sounds way better. Changed, thanks!